gef.py

```diff
@@ -6886,7 +6886,7 @@ class ProcessListingCommand(GenericCommand):
 
     def __init__(self):
         super(ProcessListingCommand, self).__init__(complete=gdb.COMPLETE_LOCATION)
-        self.add_setting("ps_command", "/bin/ps auxww", "`ps` command to get process information")
+        self.add_setting("ps_command", "ps auxww", "`ps` command to get process information")
```

Comment:
Use the `which()` function to return the absolute path of `ps`. This function has the advantage of raising a `FileNotFoundError` exception in the (unlikely) `ps` is not found in the PATH.

Reply:
Where should be called?

Reply:
Something along the lines of :
```python
    def __init__(self):
        [...]
        ps = which("ps")
        self.add_setting("ps_command", "{:s} auxww".format(ps), "`ps` command to get process information")
```
would be better. If `ps` is not found in the PATH, the line `ps = which("ps")` will raise an exception that the user will know about and fix.

Reply:
done.